### PR TITLE
tidb: Add configuration item description 'committer-concurrency'

### PR DESCRIPTION
**Pull request by 7yyo**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add configuration item description `committer-concurrency`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 7yyo — COMMENTED** [review]
👀

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -316,6 +316,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：104857600 (Byte)
 + 单个事务中，所有 key-value 记录的总大小不能超过该限制。该配置项的最大值不超过 `10737418240`（表示 10GB）。注意，如果使用了以 `Kafka` 为下游消费者的 `binlog`，如：`arbiter` 集群，该配置项的值不能超过 `1073741824`（表示 1GB），因为这是 `Kafka` 的处理单条消息的最大限制，超过该限制 `Kafka` 将会报错。
 
+### `committer-concurrency`
+
++ 用于控制 retry commit 阶段的 goroutine 数量
```
Comment (cfzjywxk) [inline tidb-configuration-file.md:321]: 这里应该是 “单个事务提交阶段，用于执行提交操作相关请求的 goroutine 数量"

**Comment by 7yyo** [comment]
@cfzjywxk PTAL

**Comment by tiancaiamao** [comment]
/LGTM
@TomShawn @cfzjywxk

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -328,6 +328,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：104857600 (Byte)
 + 单个事务中，所有 key-value 记录的总大小不能超过该限制。该配置项的最大值不超过 `10737418240`（表示 10GB）。注意，如果使用了以 `Kafka` 为下游消费者的 `binlog`，如：`arbiter` 集群，该配置项的值不能超过 `1073741824`（表示 1GB），因为这是 `Kafka` 的处理单条消息的最大限制，超过该限制 `Kafka` 将会报错。
 
+### `committer-concurrency`
+
++ 单个事务提交阶段，用于执行提交操作相关请求的 goroutine 数量
```
Comment (TomShawn) [inline tidb-configuration-file.md:333]:
```suggestion
+ 在单个事务的提交阶段，用于执行提交操作相关请求的 goroutine 数量
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**